examples/simple-jsbeautifyrc/markdown/expected/test.md

```diff
@@ -1,11 +1,11 @@
 # heading 1
 
-- item
-- item
-- item
+-   item
+-   item
+-   item
 
 ## heading 2
 
-1. one
-2. two
-3. three
+1.  one
+2.  two
+3.  three
```

Comment:
Looks like it worked. However, how can a user configure this indentation? It appears to be 3 spaces for `-` bullets and 2 spaces for numbered bullets, such that they align for both. This looks nice, however for our current users this will cause a lot of whitespace reformatting when they beautify their Markdown files. If we could configure the default to work like it did previous that would be ideal.

Original: https://raw.githubusercontent.com/kachkaev/atom-beautify/18e7f5ebad5f83e25a6a03e05ee15ada93329328/examples/simple-jsbeautifyrc/markdown/original/test.md

```markdown
#            heading 1

- item
-  item
-   item

##     heading 2

1. one
2.  two
2.   three
```

Reply:
I think this option is `listItemIndent` or `incrementListMarker`: https://github.com/remarkjs/remark/tree/master/packages/remark-stringify#optionslistitemindent

Reply:
Hmm... I'm not sure we should be switching back to old indents. Yes, the existing users may experience a one-time problem after an update, but that's expected given that the formatter being used is now different. Anyone can switch back to _Tidy Markdown_ and thus restore the original behaviour.

Four characters per intent is a very reasonable choice and sticking to this rule has more benefits in the long run. In any case, it's not the only breaking change that people might see.

Reply:
In light of the lack of YAML font-matter support and these indentation changes, we need to make sure the `CHANGELOG` describes all of the breaking changes made by switching to Remark. Also, I would like to have *more* examples/tests added for Markdown showcasing the value and expected result from switching to Remark.